app/main.py

```diff
@@ -1,8 +1,35 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    info_list = []
+
+    for person in people:
+        Person(person["name"], person["age"])
```

Comment:
Could you add the created person to the list in this line 
It would help to avoid code duplication

Reply:
I supposed I must add new instances before adding persons to the list, but it was yesterday) Shorted cod will be better. Thanks!